### PR TITLE
Program to implement IP Address Type and also Stringer interface

### PR DESCRIPTION
**Pull request by eobjects**
Fixes #70 

#### Changes proposed in this PR:

- Program to implement IP Address Type and also Stringer interface
- Implemented test cases for checking Stringer interface implementation

### CODE REVIEW

File: 05_stringer/main_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var testMatrix = []struct {
+	input  IPAddr
+	output string
```
Comment (juliaogris) [inline 05_stringer/main_test.go]: I'd call this `expected` or `expectedOutput`

File: 05_stringer/main_test.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+func TestStringer(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	for _, testData := range testMatrix {
+		buf.Reset()
+		fmt.Fprint(out, testData.input)
+		r.EqualValues(testData.output, buf.String())
```
Comment (juliaogris) [inline 05_stringer/main_test.go]: No need for buf.
Just test as:
```go
for _, testData := range testMatrix {
		r.EqualValues(testData.output, testData.input.String())
}
```

**Review by juliaogris — CHANGES_REQUESTED** [review]
I've updated your PR description: `- ` (note the space after `-`) render bullets.

Looks very close.

File: 05_stringer/main_test.go (hunk 1 above)
Comment (eobjects) [reply 05_stringer/main_test.go]: fixed this

File: 05_stringer/main_test.go (hunk 2 above)
Comment (eobjects) [reply 05_stringer/main_test.go]: updated

**Review by juliaogris — APPROVED** [review]
LGTM 💚

**Pull request merged**